### PR TITLE
Remove CB tools from the untrack list in Dscript

### PR DESCRIPTION
**Pull request by norahuang**

### CODE REVIEW

File: Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc [hunk 1]
```diff
@@ -114,35 +114,16 @@ export function install(args: Arguments) : Result {
 
             const qAuthMaterial = {
                 environmentVariables: [{name: "QAUTHMATERIALROOT", value: qAuthMaterialRoot.path}],
```
Comment (dannyvv) [inline Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc]: Should we make this environment variable a passthrough variable as well then?

Comment (norahuang) [reply Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc]: yes, I think so, and it has already been marked as passthrough by GBR. I will remove this line.

File: Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc [hunk 2]
```diff
@@ -180,12 +180,9 @@ export function runQTest(args: QTestArguments): Result {
             qTestContextInfoPath,
         ],
         untrackedScopes: [
-            d`d:/data`,
-            d`d:/app`,
```
Comment (mpysson) [inline Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc]: Friendly reminder to not merge this until the GBR deployment that defaults to the session global untracking behavior

**Comment by mpysson** [comment]
>                 environmentVariables: [{name: "NUGET_CREDENTIALPROVIDERS_PATH", value: nugetCredentialProviderPath.path}],

Is this a global pass through variable as well? If not you should at least put it in passThroughEnvironmentVariables on the pip so we can get cache hits from CloudBuild builds.

---
Refers to: Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc:94 in 76b8536. [](commit_id = 76b8536fc363d2db320768b05c39a49ae561b009, deletion_comment = False)

**Comment by norahuang** [comment]
> environmentVariables: [{name: "NUGET_CREDENTIALPROVIDERS_PATH", value: nugetCredentialProviderPath.path}],
> 
> 
> Is this a global pass through variable as well? If not you should at least put it in passThroughEnvironmentVariables on the pip so we can get cache hits from CloudBuild builds.
> 
> Refers to: Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc:94 in 76b8536. [](commit_id = 76b8536, deletion_comment = False)

yes, it is in the global list as well.

File: Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc (hunk 2 above)
Comment (norahuang) [reply Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc]: Sure, this is a draft PR. I won't make it ready until GBR is ready.

Comment (norahuang) [reply Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc]: The GBR has been deployed, I am going to merge this PR.

**Pull request merged**